cmd/bootstrap/cmd/observer_network_key.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/cmd"
+	"github.com/onflow/flow-go/cmd/bootstrap/utils"
+	"github.com/onflow/flow-go/crypto"
+)
+
+var (
+	flagOutputFile string
+)
+
+// observerNetworkKeyCmd represents the `observer-network-key` command which generates required
+// network key for an Observer, and writes it to the provided path. Used by new Observer operators
+// to create the networking key only
+var observerNetworkKeyCmd = &cobra.Command{
+	Use:   "observer-network-key",
+	Short: "Generates network key and writes it to the provided path",
+	Run:   observerNetworkKeyRun,
+}
+
+func init() {
+	rootCmd.AddCommand(observerNetworkKeyCmd)
+
+	observerNetworkKeyCmd.Flags().StringVarP(&flagOutputFile, "output-file", "f", "", "output file path")
+	cmd.MarkFlagRequired(observerNetworkKeyCmd, "output-file")
+
+	observerNetworkKeyCmd.Flags().BytesHexVar(
+		&flagNetworkSeed,
+		"seed",
+		[]byte{},
+		fmt.Sprintf("hex encoded networking seed (min %d bytes)", crypto.KeyGenSeedMinLenECDSASecp256k1))
+}
+
+// observerNetworkKeyRun generate a network key and writes it to the provided file path.
+func observerNetworkKeyRun(_ *cobra.Command, _ []string) {
+
+	// generate seed if not specified via flag
+	if len(flagNetworkSeed) == 0 {
+		flagNetworkSeed = GenerateRandomSeed(crypto.KeyGenSeedMinLenECDSASecp256k1)
+	}
+
+	// if the file already exists, exit
+	keyExists, err := pathExists(flagOutputFile)
+	if err != nil {
+		log.Fatal().Err(err).Msgf("could not check if %s exists", flagOutputFile)
+	}
+
+	if keyExists {
+		log.Warn().Msgf("%s already exists, exiting...", flagOutputFile)
+		return
+	}
+
+	// generate unstaked networking private key
+	networkKey, err := utils.GenerateUnstakedNetworkingKey(flagNetworkSeed)
```

Comment:
What are "observers" exactly? are these a sub-type of unstaked access nodes?
I'm asking to make sure `GenerateUnstakedNetworkingKey` is what we need and not `GenerateNetworkingKey`.

Reply:
Observer is the new name for unstaked access node.

Reply:
ah okay 😅  It makes sense to use that function then.